### ISSUE TITLE
Issue with either bearer token, or regex?

### ISSUE THREAD

**Issue by icsy7867** [statping-ng/statping-ng#204]
Not sure what I am doing wrong.  Trying to hit an API on a confluence server for a health check, and I am looking for a:
"isHealthy": true

https://myserver.com/rest/troubleshooting/1.0/check/

This works in postman using an API token as an auth bearer.

Here is the curl equivalent
```
curl --location --request GET 'https://myserver.com/rest/troubleshooting/1.0/check/' \
--header 'Authorization: Bearer XXXXXXX' \
...
```
And it returns a JSON object:

```
{
    "statuses": [
        {

            ...
            "isHealthy": true,
            ...

```

However, unless my regex is (.*), Matches everything, it fails.  Hoping someone could offer some guidance on what proper regex for this looks like?  I can easily test things that work here https://regex101.com/

But none of these seem to work.

**Comment by almereyda** [comment]
Yes, documented examples or tests for the regex matcher are missing.

There's a link to an example of matching a JSON response in https://regex101.com/r/I5bbj9/1 hidden behind the small Regex link below the field.

https://github.com/statping-ng/statping-ng/blob/4d872facdafa1f527d5b2c46e4c09623804f5e2c/frontend/src/forms/Service.vue#L154
